cmd/qat_plugin/README.md

```diff
@@ -52,81 +224,84 @@ The number of devices discovered are:10
 device-plugin start server at: /var/lib/kubelet/device-plugins/intelQAT.sock
 device-plugin registered
 ListAndWatch: Sending device response
-
 ```
 
-By default, the device plugin supports these QuickAssist devices:  DH895xCC, C62x, C3xxx, and D15xx devices.
+## QAT device plugin Demos
 
-Use the `kernel-vf-drivers flag` to specify the vf Device Driver for the particular QAT device. For more information, refer to [Intel QAT Crypto Poll Mode Driver](https://dpdk.org/doc/guides/cryptodevs/qat.html).
+The below sections cover a `DPDK` and `OpenSSL` demo, both of which utilise the
+QAT device plugin under Kubernetes.
 
-`-dpdk-driver` is set to `vfio-pci` by default since it is more robust and secure driver compared with `igb_uio`. See [DPDK Linux Driver Guide](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html) for more information.
+### DPDK QAT demo
 
-`-mode` parameter can be set to `kerneldrv` to override the default  mode (`dpdkdrv`). With `-mode kerneldrv`, no other parameter documented above are valid, except `-debug` which is global for both modes. `kerneldrv` mode implements resource allocation based on system configured [logical instances](https://01.org/sites/default/files/downloads//336210-009qatswprogrammersguide.pdfhttps://01.org/sites/default/files/downloads//336210-009qatswprogrammersguide.pdf). The mode does not guarantee full device isolation between containers and therefore it's not recommended. Moreover, the mode will be deprecated and removed once `libqat` implements non-UIO based device access.
+#### DPDK Prerequisites
 
-### Build QAT device plugin Docker image:
-```
-$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
-$ make  intel-qat-plugin
-```
+For the DPDK QAT demo to work, the Data Plane Development Kit (DPDK) drivers must be loaded and configured.
+For more information, refer to:
+[DPDK Getting Started Guide for Linux](https://doc.dpdk.org/guides/linux_gsg/index.html) and
+[DPDK Getting Started Guide, Linux Drivers section](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
 
-**Note**: `kerneldrv` mode is excluded from the build by default. Add `BUILDTAGS=kerneldrv` to `make` to get `kerneldrv` functionality added to the build.
+#### Build the image
 
-### Deploy QAT device plugin as a DaemonSet:
-```
-$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
-kubectl create -f deployments/qat_plugin/qat_plugin_default_configmap.yaml
-kubectl create -f deployments/qat_plugin/qat_plugin.yaml
+First you need to build the DPDK demo image:
```

Comment:
this is also provided as pre-built image. https://hub.docker.com/r/intel/crypto-perf